cmd/driver/driver_linux.go

```diff
@@ -177,12 +178,23 @@ func NewDriverCmd(ctx context.Context, opt *options.Common) *cobra.Command {
 func loadDriverVersion() string {
 	isSet := false
 	greatestVrs := semver.Version{}
-	paths, _ := filepath.Glob("/usr/src/falco-*+driver")
+	paths, _ := filepath.Glob("/usr/src/falco-*")
 	for _, path := range paths {
+		fileInfo, err := os.Stat(path)
+		// We expect path to point to a folder,
+		// otherwise skip it.
+		if err != nil {
+			continue
+		}
+		if !fileInfo.IsDir() {
+			continue
+		}
 		drvVer := strings.TrimPrefix(filepath.Base(path), "falco-")
 		sv, err := semver.Parse(drvVer)
 		if err != nil {
-			continue
+			// Not a semver; return it because we
+			// Won't be able to check it against semver driver versions.
+			return drvVer
```

Comment:
In case found driver version is not semver, we cannot check it against any other semver driver version.
Return it as is.

Reply:
In any case, it is not expected for the user to have multiple `falco-*` folders under `/usr/src`.